src/heart.c

```diff
@@ -160,6 +161,15 @@ static int wdt_timeout = 60;
 /* last_wdt_pet_time is the absolute time that hardware watchdog was pet */
 static time_t last_wdt_pet_time = 0;
 
+/* Timeout on receiving a handshake message from the application that the heart callback. 0=unused */
```

Comment:
```suggestion
/* Timeout on receiving a handshake message from the application that the heart callback was set. 0=unused */
```
I believe this is what you meant?